types/src/transaction/transaction_v1/transaction_v1_header.rs

```diff
@@ -151,6 +151,21 @@ impl TransactionV1Header {
         self.timestamp.saturating_add(self.ttl)
     }
 
+    /// Returns the gas tolerance for the given transaction.
+    pub fn gas_tolerance(&self) -> u64 {
+        match self.pricing_mode {
+            PricingMode::Classic { gas_price, .. } => gas_price,
+            PricingMode::Fixed {
+                gas_price_tolerance,
+                ..
+            } => gas_price_tolerance,
+            PricingMode::Reserved { .. } => {
+                // TODO: Change this when reserve gets implemented.
+                0u64
```

Comment:
i think 0 actually is the correct value...they already paid so should have no tolerance for paying more / again?

Reply:
Okay, good, I just put the TODO as a reminder to address this